h/groups/test/views_test.py

```diff
@@ -349,6 +349,125 @@ def test_read_if_already_a_member_returns_response(Group, renderers):
     assert views.read(request) == mock.sentinel.response
 
 
+@read_fixtures
+def test_read_calls_search(Group, search, renderers):
```

Comment:
Small quibble on the name. We're not testing that it calls `search`. We're testing the collaboration between the view function and the `search` function -- that is, we're testing that it calls `search` with the correct arguments. Might be nice to make that clearer in the method name or the docstring, or both.